llama_index/storage/storage_context.py

```diff
@@ -45,7 +49,7 @@ class StorageContext:
 
     docstore: BaseDocumentStore
     index_store: BaseIndexStore
-    vector_store: VectorStore
+    vector_stores: Dict[str, VectorStore]
     graph_store: GraphStore
```

Comment:
wondering that do we need have separate index, graph, doc stores in the future

Reply:
yea we do acutally, since they all serve very different and distinct purposes 👍🏻